R_Friedmantest.qmd

```diff
@@ -0,0 +1,61 @@
+---
+title: "Friedman Chi-Square test using R package rstatix"
+---
+
+## R package version
+
+R 4.3.1\
+rstatix 0.7.2
+
+## Data used
+
+Simulated dataset of 10 subjects(blocks) with continuous endpoints are generated for single-drug repeated measurements to check whether any significance exists between the responses(y) at different time points(4 time points simulated)(groups). The p-value will indicate whether differences in response for different time points are significant.
+
+## Data source
+
+```{r eval=TRUE}
+library(dplyr)
+library(rstatix) 
+```
+
+```{r eval=TRUE}
+set.seed(123)
+
+One_way_repeat <- data.frame(subject = rep(1:10, each=4),
```

Comment:
I wouldn't recommend that you use a capital letter for your variable name. Of course this doesn't affect the quality and is a bit pedantic :)

Reply:
Thanks for all the review and comments. Have updated and pushed again.